content/en/real_user_monitoring/browser/collecting_browser_errors.md

```diff
@@ -160,6 +160,26 @@ try {
 {{% /tab %}}
 {{< /tabs >}}
 
+## Troubleshooting
+
+### Fix "Script error."
```

Comment:
```suggestion
### Fixing script errors
```

Reply:
"Script error." is a well-known JavaScript error. See this [blog article](https://blog.sentry.io/2016/05/17/what-is-script-error) for instance. Can we agree on `Fixing "Script error."`?

Reply:
Yes, `Fixing "Script error." is fine with me.